docs/index.md

```diff
@@ -48,6 +48,10 @@ Databricks SQL
 * Manage [dashboards](resources/sql_dashboard.md) and their [widgets](resources/sql_widget.md).
 * Provide [global configuration for all SQL Endpoints](docs/resources/sql_global_config.md)
 
+MLFlow
+* Create [MLFlow models](resources/mlflow_models.md).
```

Comment:
1. rename `mlflow_models.md` to `mlflow_model.md`
2. also add an entry to README.md